### PR TITLE
NAS-113216 / 22.12 / NAS-113216: Hide 'acltype' and 'aclmode' when creating or editing first level dataset

### PR DESCRIPTION
**Pull request by bvasilenko**
**Summary**

Removed **ACL Type** and **Mode**,  if dataset is a **_root dataset_** , as suggested by the ticket's description

### CODE REVIEW

File: src/app/pages/datasets/components/dataset-form/dataset-form.component.ts [hunk 1]
```diff
@@ -1452,6 +1454,10 @@ export class DatasetFormComponent implements FormConfiguration {
     return false;
   }
 
+  isRootDataset(path: string): boolean {
```
Comment (undsoft) [inline src/app/pages/datasets/components/dataset-form/dataset-form.component.ts:1457]: we have `isRootDataset` in `dataset.utils.ts` that you can use here.

**Review by undsoft — CHANGES_REQUESTED** [review]
I think you misunderstood the task.
Root dataset is the first dataset in the hierarchy, it's the one that is created when you add a pool and can only be edited. It is this dataset that shouldn't have Acl Type and Acl Mode fields.
Right now this is not the case. I can still edit these fields on the root dataset, but cannot edit them on first child.

**Comment by bvasilenko** [comment]
@undsoft Done! Now it uses `isRootDataset` in `dataset.utils.ts`

**Review by undsoft — APPROVED** [review]
Works well 👍🏻

**Pull request merged**